packages/lib-classifier/src/store/AnnotationsStore.spec.js

```diff
@@ -44,4 +48,28 @@ describe('Model > AnnotationsStore', function () {
       })
     })
   })
+
+  describe(`removing an annotation`, function () {
+    const task = Task.create({ taskKey: 'T0', type: 'default', question: 'How many cats?' })
+
+    before(function () {
+      model = AnnotationsStore.create({})
+    })
+
+    describe('for a new task', function () {
+      it('should do nothing', function () {
+        model.removeAnnotation('T0')
+        expect(model.annotations.size).to.equal(0)
```

Comment:
Maybe add an `expect(model.annotations.size).to.equal(0)` before the action is called too?

Reply:
Good point. I'll do that.